openslides_backend/models/motion_change_recommendation.py

```diff
@@ -0,0 +1,45 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionChangeRecommendation(Model):
+    """
+    Model for motion change_recommendation.
+
+    Reverse fields:
+    - motion_ids
+    """
+
+    collection = Collection("motion_change_recommendation")
+    verbose_name = "motion_change_recommendation"
+
+    id = fields.IdField(description="The id of this motion change recommendation.")
+    # Replacement (0), Insertion (1), Deletion (2), Other (3)
+    type = fields.IntegerField(
```

Comment:
Just use a `PositiveIntegerField`

Reply:
Our PositiveIntegerField does not support zero. So you can use IntegerField with enum=[0,1,2,3],

Reply:
I think this has also a default value.

Reply:
Done. Use enum and set a default value to 0.

Reply:
> Our PositiveIntegerField does not support zero.
Ah, my bad.